src/main/java/com/powsybl/openloadflow/ac/outerloop/SimpleTransformerVoltageControlOuterLoop.java

```diff
@@ -23,7 +26,7 @@ public String getType() {
 
     @Override
     public void initialize(AcOuterLoopContext context) {
-        for (LfBranch controllerBranch : getControllerBranches(context.getNetwork())) {
+        for (LfBranch controllerBranch : (List<LfBranch>) context.getNetwork().getAllControllerElements(VoltageControl.Type.TRANSFORMER)) {
```

Comment:
@geofjamg the unit test is different because of that change. On main branch, we activate the transformer voltage control of all branches that have one and that are not disabled. Even if there is a potential conflict with a generator voltage control (sometimes it works, sometimes KLU issue). Now, with this branch, we keep only the generator. It is what happens at first resolution, before contingency.